network/p2p/conduit/conduit.go

```diff
@@ -41,36 +28,8 @@ func WithMisbehaviorManager(misbehaviorManager network.MisbehaviorReportManager)
 //
 //		a new instance of the DefaultConduitFactory.
 //	 an error if the initialization of the conduit factory fails. The error is irrecoverable.
-func NewDefaultConduitFactory(alspCfg *alspmgr.MisbehaviorReportManagerConfig, opts ...DefaultConduitFactoryOpt) (*DefaultConduitFactory, error) {
-	m, err := alspmgr.NewMisbehaviorReportManager(alspCfg)
-	if err != nil {
-		return nil, fmt.Errorf("could not create misbehavior report manager: %w", err)
-	}
-	d := &DefaultConduitFactory{
-		misbehaviorManager: m,
-	}
-
-	for _, apply := range opts {
-		apply(d)
-	}
-
-	cm := component.NewComponentManagerBuilder().
-		AddWorker(func(ctx irrecoverable.SignalerContext, ready component.ReadyFunc) {
-			d.misbehaviorManager.Start(ctx)
-			select {
-			case <-d.misbehaviorManager.Ready():
-				ready()
-			case <-ctx.Done():
-				// jumps out of select statement to let a graceful shutdown.
-			}
-
-			<-ctx.Done()
-			<-d.misbehaviorManager.Done()
-		}).Build()
-
-	d.Component = cm
-
-	return d, nil
+func NewDefaultConduitFactory() *DefaultConduitFactory {
```

Comment:
don't forget to update the godocs

Reply:
https://github.com/onflow/flow-go/pull/4376/commits/5fa7257ac5d2206818ecbc0cb1a839512c5b46e9